src/services/GLSL/pixel_shader_raster.glsl

```diff
@@ -117,15 +120,19 @@ void main(void) {
         x = 1.0 - x;
     }
 
-    float cmapYVal = (float(uCmapIndex) + 0.5) / float(uNumCmaps);
-    vec2 cmapCoords = vec2(x, cmapYVal);
-
-    // Apply pixel highlight
+        // Apply pixel highlight
     if (isNaN(rawVal)) {
         outColor = uNaNColor * uNaNColor.a;
     } else if (rawVal < uPixelHighlightVal) {
         outColor = vec4(x, x, x, 1);
+    } else if (uCmapIndex == -1 || uCmapIndex >= uNumCmaps) {
```

Comment:
Do we need to define cmapCoords in here?

Reply:
maybe no... do we?